src/elements/content-sidebar/activity-feed/version/__tests__/Version.test.js

```diff
@@ -68,7 +73,7 @@ describe('elements/content-sidebar/ActivityFeed/version/Version', () => {
         ${defaultUser}      | ${defaultUser.name}
         ${restoreUser}      | ${restoreUser.name}
         ${trashedUser}      | ${trashedUser.name}
-        ${PLACEHOLDER_USER} | ${''}
+        ${PLACEHOLDER_USER} | ${(<FormattedMessage {...priorCollaborator} />)}
```

Comment:
did you have to use `FormattedMessage` here? the test below only provides the message object

Reply:
Yes because we're passing in a FormattedMessage instead of string as a placeholder if `name` is empty. `const renderName = name || <FormattedMessage {...messages.priorCollaborator} />;` . If we pass in a string, I believe `A Prior Collaborator` wouldn't be translated.

Reply:
wait `getVersionUser` doesnt return a `FormattedMessage` of "A Prior Collabrorator," why would we pass in as formatted here?